src/components/application_manager/include/application_manager/application.h

```diff
@@ -560,6 +576,18 @@ class Application : public virtual InitialApplicationData,
   virtual bool IsSubscribedToIVI(uint32_t vehicle_info_type) const = 0;
   virtual bool UnsubscribeFromIVI(uint32_t vehicle_info_type) = 0;
 
+#ifdef SDL_REMOTE_CONTROL
+  /**
+   * @brief Return pointer to extension by uid
+   * @param uid uid of extension
+   * @return Pointer to extension, if extension was initialized, otherwise NULL
+   */
+  virtual AppExtensionPtr QueryInterface(AppExtensionUID uid) = 0;
+  virtual bool AddExtension(AppExtensionPtr extention) = 0;
+  virtual bool RemoveExtension(AppExtensionUID uid) = 0;
+  virtual void RemoveExtensions() = 0;
+  virtual const std::set<uint32_t>& SubscribesIVI() const = 0;
+#endif
```

Comment:
@LuxoftAKutsan , @LevchenkoS  , maybe it is good to move all rsdl funcs in file to one scope